web/src/components/TaskStatusChangeSnackbar.tsx

```diff
@@ -5,13 +5,26 @@ import { ReactElement } from "react";
 interface Props {
   isOpen: boolean;
   close: () => void;
+  taxRegistrationSnackbarIsOpen: boolean;
+  closeTaxRegistrationSnackbar: () => void;
```

Comment:
[Pebble] Are these referencing the state of this component? If that's the case why not just rename them something a little simpler?
```suggestion
  isOpen: boolean;
  closeSnackbar: () => void;
```

Reply:
If this is self-referencing I'm going to say this should be a [Boulder] – the component we're talking about is actually called `TaskStatusChangeSnackbar` and this variable seems to be talking about another component that I don't think actually exists.